src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/FormatedStringPartConverter.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Extracts from a formated string with one '{0}' placeholder to either left or right side of the placeholder
+    /// </summary>
+    public class FormatedStringPartConverter : IValueConverter
+    {
+        /// <summary>
+        /// Extracts either left or right side of a string with one placeholder '{0}'
+        /// </summary>
+        /// <param name="value">string with placeholder</param>
+        /// <param name="targetType">Not used</param>
+        /// <param name="parameter">0 for left side, 2 for right side</param>
```

Comment:
The mode in this class is a bit difficult to understand. Why are the values 0 and 2? Could we consider having a string parameter that reflects the operation instead?

Reply:
I prefer int's because making a int comparison only requires 1 machine operation. For strings, I imagine you need to iterate over string characters to compare for equality.

Reply:
I am considering changing to an enum or a string.

Reply:
Added enum.